packages/trader/src/Modules/Trading/Components/Form/ContractType/ContractTypeInfo/contract-type-info.jsx

```diff
@@ -39,23 +68,47 @@ const Info = ({ handleNavigationClick, handleSelect, initial_index, item, list }
     });
 
     return (
-        <Carousel
-            className='contract-type-info'
-            bullet_color='var(--text-disabled)'
-            active_bullet_color='var(--brand-red-coral)'
-            initial_index={initial_index}
-            onItemSelect={active_index => {
-                handleNavigationClick(contract_types[active_index]);
-            }}
-            list={cards}
-            width={isMobile() ? 328 : 290}
-        />
+        <React.Fragment>
+            {(carousel_index === 8 || carousel_index === 1) && (
+                <div className='contract-type-info__button-wrapper'>
+                    <ButtonToggle
+                        buttons_arr={[
+                            { text: localize('Description'), value: 'description' },
+                            { text: localize('Glossary'), value: 'glossary' },
+                        ]}
+                        name='description_glossary_filter'
+                        is_animated
+                        has_rounded_button
+                        onChange={e => {
+                            setSelectedTab(e.target.value);
+                        }}
+                        value={selected_tab}
+                    />
+                </div>
+            )}
+            <Carousel
+                className='contract-type-info'
+                bullet_color='var(--text-disabled)'
+                active_bullet_color='var(--brand-red-coral)'
+                initial_index={carousel_index}
+                onItemSelect={active_index => {
+                    setCarouselIndex(active_index);
```

Comment:
Function, passed as `onItemSelect` is not small, may be it'll be better to declare this function upstairs? Before `return`